pkg/utils/host.go

```diff
@@ -0,0 +1,134 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/glog"
+
+	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+)
+
+const (
+	SriovConfBasePath     = "/etc/sriov-operator"
```

Comment:
It's duplicated with https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/37ddcd4fd3c011241348bd6a3efce9d426b07f6a/pkg/utils/sriov.go#L31